content/docs/command-reference/import.md

```diff
@@ -137,6 +137,38 @@ Several of the values above are pulled from the original stage file
 subfields under `repo` are used to save the origin and version of the
 dependency, respectively.
 
+### Importing from non-DVC repositories
+
+You can even import files (datasets) from Git repositories that are not
+specifically <abbr>DVC repositories</abbr>. Here's an example importing a
+dataset from [GSA's data repo](https://github.com/GSA/data).
```

Comment:
Please
- ~~(datasets)~~
- Git repositories -> plain Git repos
- ~~specifically~~
- Here's an example importing a -> For example, let's import a

Reply:
Question: does it make sense why I'm asking for these small wording changes? Can you maybe tell me why, if so? Just checking if we can be on a close-enough page in terms of writing style 🙂

Reply:
- [ ] Also here: please clarify that you can also import Git-tracked files from DVC repos. Meaning the repo is tracked both by Git and DVC but you can import any file regardless.

Reply:
>     * [ ]  Also here: please clarify that you can also import Git-tracked files from DVC repos. Meaning the repo is tracked both by Git and DVC but you can import any file regardless.

Do you think this point would be more suited to add in the section where we are importing from a DVC repository? Because it will make more sense to tell the readers then about the files they can import from such a repository. Just an opinion here.

Reply:
> Question: does it make sense why I'm asking for these small wording changes? Can you maybe tell me why, if so? Just checking if we can be on a close-enough page in terms of writing style slightly_smiling_face

Sure. Documentations are important for users as much as the product. It's important to maintain precision with the semantics and vocabulary since users often hunt through the documentation to find a specific issue.

1. ~~(datasets)~~ : Might confuse the reader if the usage of this example(or even import) is limited to datasets and not any files in general. Of course, using datasets on my part itself was a silly assumption that I made which I shouldn't have.

2. "plain Git repos" : Better simpler language to highlight non-DVC repos

3. ~~specifically~~ : Didn't make much sense to use this word now that I re-read the documentation.

4. "Let's import..." : Better simpler language for the readers that puts the statement in an encouraging tone rather than sounding like a Wikipedia article.

Reply:
Agree on terminology, thanks for the answer 🙂 

> > please clarify that you can also import Git-tracked files from DVC repos.
>
> Do you think this point would be more suited to add in the section where we are importing from a DVC repository?

Hmmm what section is that? The other 2 examples are for other specific scenarios. Maybe in the description, yes. But I still think it's suitable to clarify it here in this example, just a note like "> Note that Git-tracked files can be imported from DVC repos as well."